.travis.yml

```diff
@@ -1,44 +1,78 @@
-language: node_js
-node_js:
-- '0.10'
-- '0.11'
-- '0.12'
-compiler: clang
+os:
+- linux
+- osx
+# turn on new container-based infrastructure
+sudo: false
+language: cpp
+addons:
+  apt:
+    sources:
+    - ubuntu-toolchain-r-test
+    packages:
+    - g++-4.8
+    - g++-4.8-multilib
+    - gcc-multilib
 env:
+  matrix:
+  # - TRAVIS_NODE_VERSION="0.8"
+  - TRAVIS_NODE_VERSION="0.10"
+  - TRAVIS_NODE_VERSION="0.12"
+  - TRAVIS_NODE_VERSION="iojs-3"
```

Comment:
I copied this part from Nan, which tested against iojs-3. If you don't like, I can remove it.